yarn-project/end-to-end/src/e2e_token_contract/minting.test.ts

```diff
@@ -40,17 +40,11 @@ describe('e2e_token_contract minting', () => {
       });
 
       it('mint >u128 tokens to overflow', async () => {
-        const maxAmountWithoutOverflow = 2n ** 128n - 1n - tokenSim.balanceOfPublic(accounts[0].address);
+        const overflowAmount = 2n ** 128n;
 
-        // First we send a valid tx because if we minted with "amount > U128::max()" we would get an error in U128
-        // in encoder.ts
-        await asset.methods.mint_to_public(accounts[0].address, maxAmountWithoutOverflow).send().wait();
-        tokenSim.mintPublic(accounts[0].address, maxAmountWithoutOverflow);
-
-        // Then we try to mint 1 to cause the U128 overflow inside the contract
-        await expect(asset.methods.mint_to_public(accounts[0].address, 1n).simulate()).rejects.toThrow(
-          U128_OVERFLOW_ERROR,
-        );
+        await expect(
+          asset.methods.mint_to_private(accounts[0].address, accounts[0].address, overflowAmount).simulate(),
+        ).rejects.toThrow('Cannot satisfy constraint');
```

Comment:
Since this is a private function this issue does not apply here and we get a proper check on the input arg. The error message is like that as the `U128_OVERFLOW_ERROR` used to be on the U128 type and it's no longer present on the `u128` one.